testdata/bless_provider/scripts/component.mk

```diff
@@ -42,11 +42,16 @@ lint-terraform-fmt: terraform
 	@echo
 .PHONY: lint-terraform-fmt
 
+check-auth:
+	@aws --profile $(AWS_BACKEND_PROFILE) sts get-caller-identity >/dev/null || echo "AWS AUTH error. This component is configured to use a profile name $(AWS_BACKEND_PROFILE). Please add one to your ~/.aws/config"
+	@aws --profile $(AWS_PROVIDER_PROFILE) sts get-caller-identity >/dev/null || echo "AWS AUTH error. This component is configured to use a profile name $(AWS_PROVIDER_PROFILE). Please add one to your ~/.aws/config"
```

Comment:
what if we have components without an aws provider configured?

Reply:
This should handle it https://github.com/chanzuckerberg/fogg/pull/366/files#diff-d1dfe7c391918134f1692e607df2c103R9 ?

Reply:
I will test on snowflake-infra

Reply:
derp nevermind, still won't work

Reply:
fixed